feat: Utilise create-discord-bot (#10013)
discordjs/discord.js: apps/guide/src/content/01-home/03-how-to-contribute.mdx

Requests may be as simple as "add an example to the [frequently asked questions](../topics/frequently-asked-questions) page", or as elaborate as "add a page regarding [sharding](../topics/sharding)". We'll do our best to fulfill all requests, as long as they're reasonable.

To make a request, simply head over to [the repository's issue tracker](https://github.com/discordjs/discord.js/issues) and [create a new issue](https://github.com/discordjs/discord.js/issues/new)! Title it appropriately, and let us know exactly what you mean inside the issue description. Make sure that you've looked around the site before making a request; what you want to request might already exist!

<Alert title="Tip" type="info">
	Remember that you can always [fork the repository](https://github.com/discordjs/discord.js/fork) and [make a pull
	request](https://github.com/discordjs/discord.js/pulls) if you want to add anything to the guide yourself!
</Alert>

> We'll also get into some of the more advanced features this guide does below. It is recommended to view the [source](https://github.com/discordjs/discord.js/blob/main/apps/guide/src/content/01-home/03-how-to-contribute.mdx) of this page to see how exactly they are to be used.

Reviewer: ```suggestion
We'll also get into some of the more advanced features this guide uses below. We recommended you have a look at the [source](https://github.com/discordjs/discord.js/blob/main/apps/guide/src/content/01-home/03-how-to-contribute.mdx) of this page to see exactly how they work.
```
